Inc/my_machine.h

```diff
@@ -28,6 +28,7 @@
 //#define BOARD_MORPHO_CNC      // For use with Nucleo-F411RE and F446RE boards. Work in progress.
```

Comment:
I commented out the odometer_enable.  As for adding a custom config, I thought that is what I did by defining BOARD_BLACKPILL the one that comes with the base was BOARD_MINI_BLACKPILL being a minimalistic implementation.

Reply:
Are the pin assignments the same in both map-files if ganging/auto squaring is disabled? If so, IMO, you should update the mini file instead of making a new as it will be less to choose from and less to maintain.

Reply:
I have readded your board map as it did not reuse the A-motor pins in the mini map as I thought, there were also some typos in driver.c as well as code to support ganged axes (no auto-squaring) was missing.

Do you plan to make a board for your mapping and if so are you going to publish it?

Reply:
Actually I just updated the mini to include my changes.   Even went as far as listing what pin was being used for what.